charts/supabase/templates/minio/volume.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if .Values.minio.enabled -}}
+{{- if .Values.minio.persistence.enabled -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "supabase.minio.fullname" . }}-pvc
+  labels:
+    {{- include "supabase.labels" . | nindent 4 }}
+  {{- with .Values.minio.persistence.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.minio.persistence.minioClassName }}
+  minioClassName: {{ .Values.minio.persistence.minioClassName }}
+  {{- end }}
+  accessModes:
+  {{- range .Values.minio.persistence.accessModes }}
+    - {{ . | quote }}
+  {{- end }}
+  resources:
+    requests:
+      minio: {{ .Values.minio.persistence.size | quote }}
```

Comment:
```
storage: {{ .Values.minio.persistence.size | quote }}
```

If I understand correctly `resources.requests.storage` is required

Reply:
Yes, it is indeed a typo from c6d3f1d5c223bf05eb71d1aad0ac3b7f866aaa71. Thanks for catching that!